hack/tests/e2e-ansible-molecule.sh

```diff
@@ -33,12 +33,22 @@ else
 fi
 KUSTOMIZE_PATH=${KUSTOMIZE} TEST_OPERATOR_NAMESPACE=default molecule test -s kind
 
-cd $TMPDIR
-KUSTOMIZE_PATH=${KUSTOMIZE}
+rm -rf $KUSTOMIZE
+cd $TMPDIR/
+rm -rf memcached-molecule-operator
+
 header_text "Test Ansible Molecule scenarios"
 pushd "${ROOTDIR}/test/ansible"
-DEST_IMAGE="quay.io/example/ansible-test-operator:v0.0.1"
-sed -i".bak" -E -e 's/(FROM quay.io\/operator-framework\/ansible-operator)(:.*)?/\1:dev/g' build/Dockerfile; rm -f build/Dockerfile.bak
-docker build -f build/Dockerfile -t "$DEST_IMAGE" --no-cache .
+
+make kustomize
+if [ -f ./bin/kustomize ] ; then
+  KUSTOMIZE="$(realpath ./bin/kustomize)"
+else
+  KUSTOMIZE="$(which kustomize)"
+fi
+
+DEST_IMAGE="quay.io/example/advanced-molecule-operator:v0.0.1"
+sed -i".bak" -E -e 's/(FROM quay.io\/operator-framework\/ansible-operator)(:.*)?/\1:dev/g' Dockerfile; rm -f Dockerfile.bak
+docker build -t "$DEST_IMAGE" --no-cache .
 load_image_if_kind "$DEST_IMAGE"
-OPERATOR_PULL_POLICY=Never OPERATOR_IMAGE=${DEST_IMAGE} TEST_CLUSTER_PORT=24443 TEST_OPERATOR_NAMESPACE=osdk-test molecule test --all
+KUSTOMIZE_PATH=$KUSTOMIZE OPERATOR_PULL_POLICY=Never OPERATOR_IMAGE=${DEST_IMAGE} TEST_OPERATOR_NAMESPACE=osdk-test molecule test
```

Comment:
The script is no longer running all tests, with `—all` removed?

Reply:
It should not run test all. The tests that need to be executed here are "molecule/test/default" and not kind. it is redundant and the kind/feature scenario is checked in verified the previous scenario. See: https://github.com/operator-framework/operator-sdk/pull/3433/files#diff-aded6410d6e48600cccc2fb400fbc86ac4fc7d2fc581385d611bc1388a90301cR76 that it is the reason for @fabianvf remove that from his PR.